test/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>edu.wgu.osmt</groupId>
+        <artifactId>osmt-parent</artifactId>
+        <version>2.6.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>edu.wgu.osmt</groupId>
+    <artifactId>osmt-api-test</artifactId>
+    <version>2.6.0-SNAPSHOT</version>
+    <description>WGU Open Skills Management Toolset</description>
```

Comment:
This was copy/pasta. It needs an accurate description. "OSMT API Tests" works pretty well